include/envoy/server/hot_restart.h

```diff
@@ -66,12 +63,22 @@ class HotRestart {
   virtual void shutdownParentAdmin(ShutdownParentAdminInfo& info) PURE;
 
   /**
-   * Tell our parent to gracefully terminate itself.
+   * Tell our parent process to gracefully terminate itself.
    */
   virtual void terminateParent() PURE;
 
   /**
-   * Shutdown the hot restarter.
+   * Merge stats_proto into stats_store, taking into account the stats values we've already
+   * seen transferred.
+   * @param stats_store the store whose stats will be updated.
+   * @param stats_proto the stats values we are updating with.
+   */
+  virtual void
+  mergeParentStats(Stats::StoreRoot& stats_store,
```

Comment:
Thoughts on moving this into a utility class somewhere or closer to where it is used? Does this actually depend on the hot restart implementation at all?

Reply:
Yes, makes sense; done.